rotary/config.yaml

```diff
@@ -1,16 +1,7 @@
 # Config file for rotary
 
-#####################
-## Sample information
-# The sample ID should ideally be short and without special characters (e.g., < 10 characters, alphanumeric and underscores only)
-sample_id: "E_coli"
-# Nanopore data
-longreads: "E_coli.fastq.gz"
-# QC'ed short reads
-# Set qc_short_r1 (and qc_short_r1) to "None" (with quotes) to skip short read polishing
-qc_short_r1: "QC_R1.fastq.gz"
-qc_short_r2: "QC_R2.fastq.gz"
-#####################
+# path to the sample tsv file
+sample_tsv: "example/path"
```

Comment:
@LeeBergstrand I wonder if it might be possible to completely separate `samples_tsv` from the config file in the long term? This might allow the config file to be more flexibly applied to different runs. I think that ATLAS somehow avoids specifying `samples_tsv` in the config, but I am not sure how. Anyway, I think the current solution is totally OK for this PR.

Reply:
Also, perhaps it would be better to specify the sample path as something like "/example/samples.tsv"? This change would:
- show that an absolute path is better than a relative path (unless you have found a consistent way to work with relative paths in the config now) -- I think the relative paths I had in the config file before were confusing
- show that the samples file name is usually `samples.tsv`

What do you think?

Reply:
@jmtsuji The new code looks for the sample.tsv in the current working directory (where the config is written). Thoughts?

Reply:
This sounds OK to me -- thanks!

Reply:
We should add this info to the CLI printout for `rotary run` for clarity at some point I suppose.